### ISSUE TITLE
Possibility to disable some core extensions

### ISSUE THREAD

**Issue by laurentj** [carew/carew#36]
I don't use all features of Carew : i create only pages, and haven't some blog post. So extensions like Feed or TagFeed are useless for me. I don't need the Toc extension also (moreover it causes the issue #35).

So it could be usefull to be able to disable some core-extensions. Or to activate them and turn some of them disable by default ?

Something like that ?

```
engine:
     core_extensions:
          toc: off
          feed: off
          otherExtensionWhichIsDisableByDefault: on
```

And the CoreExtension class takes care about these settings to add the listener to the dispatcher.

**Comment by lyrixx** [comment]
See http://carew.github.io/cookbook/configuration.html for more details.

**Comment by laurentj** [comment]
Thank you :-)

**Comment by lyrixx** [comment]
You are welcome ;) BTW I released carew 2.2.0
